content/ch-quantum-hardware/ac-Stark-shift.ipynb

```diff
@@ -46,10 +46,12 @@
     "# Importing standard Qiskit libraries and configuring account\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    stark_gate = Gate("stark", 1, [amplitude, drive_freq, control_freq])
</pre><p>Perhaps naming is misleading because this is spectroscopy + stark tone. What about <code>stark_spect_gate</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qiskit-community/qiskit-textbook/pull/1455/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>OK. I will change the name.</p>